control-occplugin.sh

```diff
@@ -0,0 +1,61 @@
+package: Control-OCCPlugin
+version: "%(commit_hash)s"
+tag:  master
+requires:
+  - FairRoot
+  - boost
+  - grpc
+  - protobuf
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - CMake
+source: https://github.com/AliceO2Group/Control
+incremental_recipe: |
+  make ${JOBS+-j $JOBS} prefix=$INSTALLROOT
+  make prefix=$INSTALLROOT install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -e
+
+LIBEXT=so
+case $ARCHITECTURE in
+    osx*)
+      [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost)
```

Comment:
You should do the same also for protobuf and grpc since they can all come from the system.